react-intro/react-intro-quiz-notes.md

```diff
@@ -0,0 +1,51 @@
+# react-intro-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is React?
+ A free and open-source front-end JavaScript library for building user interfaces based on components.
+
+- What is a React component?
+ A React component is a piece of UI (user interface) that has its own logic and appearance.
+ React components are JavaScript functions that return markup.
+
+- How do you mount a React app (root component) to the DOM?
+ Create a function for returning an HTML element with the first letter capitalized, nest the function into the main component file by adding the keywords “export default” before the declaring the main component function with the first letter capitalized, and you can add it to your HTML (main component) inside of the main component function by creating a self closing tag with the function name and first letter capitalized.
```

Comment:
Use the react-dom package to create a root attached to an HTML element (usually a div)
Call root.render() and pass it the app's main component.